admin/client/dist/js/MemberDatetimeOptionsetField.js

```diff
@@ -32,9 +32,9 @@
 
 		$('.memberdatetimeoptionset .toggle').entwine({
 			onclick: function onclick(e) {
-				jQuery(this).closest('.form__field-description').parent().find('.toggle-content').toggle();
+				jQuery(this).closest('.description').parent().find('.toggle-content').toggle();
```

Comment:
This regresses #5946 (which was added in the wrong file) and #5955 (which adds it in the right file, but doesn't include a compilation) - is it a deliberate change?